code/obj/window.dm

```diff
@@ -31,6 +31,8 @@ ADMIN_INTERACT_PROCS(/obj/window, proc/smash)
 	var/reinf = 0 // cant figure out how to remove this without the map crying aaaaa - ISN
 	var/deconstruct_time = 1 SECOND
 	var/image/connect_image = null
+	var/image/damage_image = null
+	var/damage_severity = null //used to determine which damage overlay to use if using them
```

Comment:
this probably doesn't need to be a var since it's recalculated every time?

Reply:
I moved the var down to the area where it's actually used (doesn't make sense to be with the others that are used *everywhere*) but I like using it as a variable cuz I can't think of a way to do it that's more readable (though it probably exists!)

Reply:
wait no figured it out like instantly after leaving this comment and changed it from using a variable to Not 👍